README.md

```diff
@@ -80,22 +80,25 @@ cardano-hw-cli node issue-op-cert
 --hw-signing-file FILE                               Input filepath of the hardware wallet signing file.
 ```
 
-## Governance voting registration
+## CIP36 voting registration
 ```
-cardano-hw-cli governance voting-registration-metadata
+cardano-hw-cli vote registration-metadata
 --mainnet | --testnet-magic NATURAL    Use the mainnet magic id or specify testnet magic id.
---vote-public-key FILE                 Input filepath to vote public key in ed25519extended format (one or more keys can be provided).
+--vote-public-key-jcli FILE            Input filepath to vote public key in ed25519extended format (one or more keys can be provided).
+--vote-public-key-string BECH32STRING  Bech32-encoded vote public key (one or more keys can be provided).
+--vote-public-key-hwsfile FILE         Input filepath to vote public key in hw-signing-file format (one or more keys can be provided).
+--vote-public-key-file FILE            Input filepath to vote public key in cardano-cli file format (one or more keys can be provided).
 --vote-weight WEIGHT                   Voting power weight assigned to vote public key.
 --stake-signing-key FILE               Input filepath of the hardware wallet stake signing file, which will be used to to sign the voting registration.
---reward-address REWARDADDRESS         Staking address which will receive voting rewards.
+--payment-address PAYMENTADDRESS       Address which will receive voting rewards.
 --nonce NONCE                          Current slot number.
 --voting-purpose VOTINGPURPOSE         Voting purpose (optional)
---reward-address-signing-key FILE      Input filepath of the reward address signing files.
+--payment-address-signing-key FILE     Input filepath of the reward address signing files.
```

Comment:
A forgotten occurence of `reward address`?

Reply:
I think this is still there (`Input filepath of the reward address signing files.`).